fahari/mle/models.py

```diff
@@ -0,0 +1,85 @@
+from django.contrib.gis.db import models
+from django.urls.base import reverse
+from django.utils import timezone
+
+from fahari.common.models import AbstractBase, Facility
+
+
+class FacilityNetworkCondition(AbstractBase):
+    """Facility network conditions."""
+
+    facility = models.ForeignKey(Facility, on_delete=models.PROTECT)
+    TRUE = "TRUE"
+    FALSE = "FALSE"
+    STATE_INDICATORS = [
```

Comment:
Use choice enumeration types instead here. Check the [docs](https://docs.djangoproject.com/en/3.2/ref/models/fields/#enumeration-types).

Reply:
fixed